app/routes/events/components/EventAttendeeStatistics.tsx

```diff
@@ -49,14 +49,21 @@ const PieChartWithLabel = ({
 }) => {
   return (
     <>
-      <h4>{label}</h4>
-      <Flex alignItems="center" style={{ marginBottom: '3rem' }} wrap={true}>
-        <DistributionPieChart
-          dataKey="count"
-          distributionData={distributionData}
-        />
-        <ChartLabel distributionData={distributionData} />
-      </Flex>
+      <div className={styles.chartCard}>
```

Comment:
Please use the Card component instead of a custom styling😁

Reply:
Of course, will do!